geomesa-arrow/geomesa-arrow-jts/src/main/java/org/locationtech/geomesa/arrow/jts/impl/AbstractPointVector.java

```diff
@@ -76,8 +87,13 @@ public void transfer(int fromIndex, int toIndex, GeometryVector<Point, FixedSize
       ((FixedSizeListVector) typed.vector).setNull(toIndex);
     } else {
       ((FixedSizeListVector) typed.vector).setNotNull(toIndex);
-      typed.writeOrdinal(toIndex * 2, readOrdinal(fromIndex * 2));
-      typed.writeOrdinal(toIndex * 2 + 1, readOrdinal(fromIndex * 2 + 1));
+      if (isFlipAxisOrder()) {
```

Comment:
I think we'd want to check if the axis order is _different_ on the destination vector here:
```suggestion
      if (isFlipAxisOrder() != typed.isFlipAccessOrder()) {
```
alternatively, i think the transfer method assumes that the source and destination vectors are the same, so maybe we don't need any logic here about axis order.